### DISCUSSION TITLE
Can we change theme?

### DISCUSSION THREAD

**Discussion by santosh** [JordanKnott/taskcafe#116] [Ideas]
Hi, today I stumbled upon this repo and cloned the repo to run the project. 

I quite did not like the dark blue shade. On the other hand, I am liking the GitHub's dimmed theme. But I don't find any mechanism to switch themes. I know I can change the CSS manually, but is ability to switch theme a good idea.

![image](https://user-images.githubusercontent.com/1515991/113494522-9b227080-9506-11eb-91b9-6d02ef097502.png)

**Comment by wyatt-howe** [comment]
For anyone in the future who wants to change the theme:

I have found you can edit the theme file here: [`frontend/src/App/ThemeStyles.ts`](https://github.com/JordanKnott/taskcafe/blob/998db2/frontend/src/App/ThemeStyles.ts).

(That image is confusing and, to clarify, is not part of taskcafe but rather GitHub.)
